content/engine/reference/commandline/scout_cache.md

```diff
@@ -13,4 +13,4 @@ repository on GitHub:
 https://github.com/docker/scout-cli
```

Comment:
Wasn't sure about this one, in the [deploy preview](https://deploy-preview-18073--docsdocker.netlify.app/engine/reference/commandline/scout_cache/) I just see the title and the description, should there be any other content here?

Reply:
Confirmed via Slack it's supposed to look this way 😎